Findexternpro.cmake

```diff
@@ -7,24 +7,18 @@ function(getCompilerPrefix _ret)
   set(options GCC_TWO_VER)
   cmake_parse_arguments(X "${options}" "" "" ${ARGN})
   if(MSVC)
-    if(MSVC14)
+    if(MSVC_VERSION GREATER 1910 AND MSVC_VERSION LESS 1919) # VS 15.0 2017
```

Comment:
this file should be a copy/paste from externpro, so changes to this file are a result of updating it to match changes in externpro.